### PR TITLE
Make app router pages smaller by reducing translations size in global 404 page

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Make root 404 page much smaller by only providing `common` translation namespace.  This page is added to HTML of every page on the site.

Before: /se - 255 KB uncompressed
After: /se - 227 KB uncompressed

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4360?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4360** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4360?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4356** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4356?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alebedev and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/appComponents/StoreLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import type { ReactNode } from 'react'
+import { Suspense } from 'react'
```
Comment (guilhermespopolin) [inline apps/store/src/appComponents/StoreLayout.tsx:2]: Not a big deal but I thought we had lint rules to force import like this in a single line, no? 🤔 
 
```suggestion
import { Suspense, type ReactNode } from 'react'
```

Comment (alebedev) [reply apps/store/src/appComponents/StoreLayout.tsx:2]: Sure, I forgot to change our eslint rules. Will do in separate PR

**Comment by alebedev** [comment]
### Merge activity

* **Jun 12, 7:16 AM EDT**: @alebedev started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4360).

**Pull request merged**